nixos/modules/services/networking/gdomap.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gdomap;
+in
+{
+  #
+  # interface
+  #
+  options = {
+    services.gdomap = {
+      enable = mkOption {
+        default = false;
+	description = "
+	  Whether to enable gdomap, the GNUstep distributed objects daemon.
+
+	  Note that gdomap runs as root.
+        ";
+      };
+
+      pidfile = mkOption {
```

Comment:
Is there a good reason for making this configurable? It is desirable to use the `RuntimeDirectory` directive and place the pid file there, if possible, or simply at the `/run` top-level if it needs to be readable by others for some reason.